atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -607,6 +608,12 @@ def get_metadata(self):
         buildroot = self.get_buildroot(build_id=self.build_id)
         output_files = self.get_output(buildroot['id'])
 
+        for output in output_files:
+            if output.metadata['buildroot_id'] == buildroot['id'] and \
+               output.metadata['type'] == 'docker-image':
+                buildroot['extra']['docker']['autorebuild'] = is_rebuild(self.workflow)
```

Comment:
We don't want to change anything in `buildroot` at all. It's the `output` object we want to set `extra.docker.autorebuild` on.